homeassistant/components/home_connect/strings.json

```diff
@@ -21,6 +22,129 @@
       "default": "[%key:common::config_flow::create_entry::authenticated%]"
     }
   },
+  "entity": {
+    "binary_sensor": {
+      "bsh-common-status-remote_control_active": {
+        "name": "Remote control"
+      },
+      "bsh-common-status-remote_control_start_allowed": {
+        "name": "Remote start"
+      },
+      "refrigeration-common-status-door-chiller_common": {
+        "name": "Chiller door alarm"
+      },
+      "refrigeration-common-status-door-freezer": {
```

Comment:
I think we should make the strings more generic. as in, `bsh-common-status-remote_control_active` can just be `remote_control_active`

Reply:
Yeah, seems reasonable, I liked the idea of having the keys from the API docs replicated right here, but it is true that it gets a little bit annoying.